src/core_plugins/kibana/public/home/components/tutorial/tutorial.js

```diff
@@ -104,6 +117,50 @@ export class Tutorial extends React.Component {
     });
   }
 
+  checkInstructionSetStatus = async (instructionSetIndex) => {
+    const instructions = this.getInstructions();
+    const esHitsCheckConfig = _.get(instructions, `instructionSets[${instructionSetIndex}].statusCheck.esHitsCheck`);
+    const { hasFailed, isComplete } = await this.fetchEsHitsStatus(esHitsCheckConfig);
+
+    this.setState((prevState) => {
```

Comment:
Any reason to ever call setState if there is no esHitsCheck?  Maybe pull out the inner 
```
+    if (!esHitsCheckConfig) {
+      return { hasFailed: false, isComplete: false };
+    }
```
so we can skip the setState call if it doesn't exist.